### PR TITLE
Daily e2e tests always run cluster-wide (i.e. not one-namespace)

### PR DESCRIPTION
**Pull request by Tahler**
Addresses #795. Will only work against istio-releases/daily-release#532.

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -556,83 +556,15 @@ presubmits:
...
-    agent: kubernetes
-    context: prow/daily-e2e-rbac-auth.sh
-    always_run: true
-    rerun_command: "/test daily-e2e-rbac-auth"
-    trigger: "(?m)^/(retest|test (all|daily-e2e-rbac-auth))?,?(\\s+|$)"
+    rerun_command: "/test daily-e2e"
+    trigger: "(?m)^/(retest|test (all|daily-e2e))?,?(\\s+|$)"
     branches:
     - master
+    - release-0.6
```
Comment (rkpagadala) [inline prow/config.yaml]: why is this required?
what happens when we run for  branch release-0.8 here

File: prow/config.yaml [hunk 2]
```diff
@@ -648,14 +580,15 @@ presubmits:
...
-      context: prow/daily-e2e-rbac-auth-skew.sh
+      context: prow/daily-e2e-skew.sh
       always_run: true
-      rerun_command: "/test daily-e2e-rbac-auth-skew"
-      trigger: "(?m)^/(retest|test (all|daily-e2e-rbac-auth-skew))?,?(\\s+|$)"
+      rerun_command: "/test daily-e2e-skew"
+      trigger: "(?m)^/(retest|test (all|daily-e2e-skew))?,?(\\s+|$)"
       branches:
       - master
+      - release-0.6
```
Comment (rkpagadala) [inline prow/config.yaml]: why is this required?
what happens when we run for  branch release-0.8 here

File: prow/config.yaml [hunk 3]
```diff
@@ -671,14 +604,15 @@ presubmits:
...
-    context: prow/daily-e2e-rbac-auth-default.sh
+    context: prow/daily-e2e-default.sh
     always_run: true
-    rerun_command: "/test daily-e2e-rbac-auth-default"
-    trigger: "(?m)^/(retest|test (all|daily-e2e-rbac-auth-default))?,?(\\s+|$)"
+    rerun_command: "/test daily-e2e-default"
+    trigger: "(?m)^/(retest|test (all|daily-e2e-default))?,?(\\s+|$)"
     branches:
     - master
+    - release-0.6
```
Comment (rkpagadala) [inline prow/config.yaml]: same here

**Review by sebastienvas — CHANGES_REQUESTED** [review]
as I commented in https://github.com/istio-releases/daily-release/pull/532, the PR does not make much sense, we should not be renaming anything, but instead removing the cluster wide one and make the cluster wide default since that s the intent of #795.

File: prow/config.yaml (hunk 1 above)
Comment (Tahler) [reply prow/config.yaml]: I'm unsure if or why they are needed. That line was present in the configurations for the daily-e2e-cluster_wide tests and I think I miscommunicated with Seb, thinking we should keep them.

**Review by sebastienvas — APPROVED** [review]
